django/forms/renderers/templates.py

```diff
@@ -0,0 +1,61 @@
+import os
```

Comment:
@prestontimmons, Just to be sure that the module structure isn't needlessly complicated, I was wondering if you envisioned other renders in other files in this module at some point?